src/index.test.ts

```diff
@@ -447,7 +447,7 @@ test.each([
   expect(zer).toEqual(shape);
   expect(zerialize(dezerialize(shape) as any)).toEqual(zerialize(schema));
   const parsed = zodexSchema.safeParse(shape);
-  expect(parsed.success).to.be.true;
+  expect(parsed.success).to.equal(true);
```

Comment:
nit: I'm not against (nor for) it, but I am curious about the motivation

Reply:
Was added as a requirement by typescript-eslint...